src/pages/policies/policies.tsx

```diff
@@ -65,24 +65,51 @@ export default function Policies() {
     );
   }
 
-  return (
-    <PoliciesLayout>
-      {filteredPolicies.length > 0 ? (
-        <>
-          <div className={styles.resultsInfo}>
-            {filteredPolicies.length} result(s)
-            {query && <> for "{query}"</>}
-          </div>
-          <PolicyList policies={pagedPolicies} />
-          <Pagination totalResults={filteredPolicies.length} currentPage={currentPage} className={styles.pagination} />
-        </>
-      ) : policies.length === 0 ? (
-        <p className={styles.emptyState}>There are no policies linked to your account.</p>
-      ) : (
+  if (!filteredPolicies.length) {
+    let policyQualifier;
+    if (filter === 'active') {
+      policyQualifier = <span className={styles.highlight}>active </span>;
+    } else if (filter === 'inactive') {
+      policyQualifier = <span className={styles.highlight}>inactive </span>;
+    }
+
+    return (
+      <PoliciesLayout>
         <p className={styles.emptyState}>
-          {query ? <>There are no policies matching "{query}"</> : <>There are no matching policies.</>}
+          {policies.length === 0 ? (
+            <>
+              You don't have any policies associated with the connected address.
+              <br />
+              Connect an address associated with a policy to start a claim.
+            </>
+          ) : filter === 'none' ? (
+            <>Please select a filter.</>
+          ) : query ? (
+            <>
+              We couldn't find any {policyQualifier}policies with <span className={styles.highlight}>"{query}"</span>.
+              <br />
+              Please try a different search term.
+            </>
+          ) : (
+            <>
+              You don't have any {policyQualifier}policies associated with the connected address.
+              <br />
+              Connect an address associated with a policy to start a claim.
+            </>
+          )}
         </p>
-      )}
+      </PoliciesLayout>
+    );
+  }
+
+  return (
+    <PoliciesLayout>
+      <div className={styles.resultsInfo}>
+        {filteredPolicies.length} result(s)
+        {query && <> for "{query}"</>}
```

Comment:
Just for clarification: I am completely OK with a space here since you are building up a string, but I think it's better to use a CSS spacing for standalone texts (which I commented above).